packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/KeyValuesTable.js

```diff
@@ -16,6 +16,7 @@
 
 import React from 'react';
 import jsonMarkup from 'json-markup';
+import { Dropdown, Icon, Menu } from 'antd';
```

Comment:
This file looks great! Awesome.

(Not sure how to add a comment to a file instead of a line...)

Reply:
Thank you for your encouraging feedback!